packages/neutron-sdk/src/stargate/msg_dex.rs

```diff
@@ -0,0 +1,44 @@
+use crate::stargate::aux::create_stargate_msg;
+use crate::stargate::proto_types::neutron::dex::{
+    MsgCancelLimitOrder, MsgDeposit, MsgMultiHopSwap, MsgPlaceLimitOrder,
+    MsgWithdrawFilledLimitOrder, MsgWithdrawal,
+};
+use crate::stargate::types_dex::{
+    CancelLimitOrderRequest, DepositRequest, MultiHopSwapRequest, PlaceLimitOrderRequest,
+    WithdrawFilledLimitOrderRequest, WithdrawalRequest,
+};
+use cosmwasm_std::CosmosMsg;
+
+const DEPOSIT_MSG_PATH: &str = "/neutron.dex.MsgDeposit";
+const WITHDRAWAL_MSG_PATH: &str = "/neutron.dex.MsgWithdrawal";
+const PLACE_LIMIT_ORDER_MSG_PATH: &str = "/neutron.dex.MsgPlaceLimitOrder";
+const WITHDRAW_FILLED_LIMIT_ORDER_MSG_PATH: &str = "/neutron.dex.MsgWithdrawFilledLimitOrder";
+const CANCEL_LIMIT_ORDER_MSG_PATH: &str = "/neutron.dex.MsgCancelLimitOrder";
+const MULTI_HOP_SWAP_MSG_PATH: &str = "/neutron.dex.MsgMultiHopSwap";
+
+pub fn msg_deposit(req: DepositRequest) -> CosmosMsg {
```

Comment:
What do you think of adding doc comments for each method which describes what does it do?